release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1012,11 +1017,16 @@ error: the server doesn't have a resource type "operators"
 
 Your upgrade to {product-title} 4.6 should now no longer be blocked by this feature.
 
-[id="ocp-4-6-noe-maintenance-operator-validation"]
+[id="ocp-4-6-node-maintenance-operator-validation"]
 ==== Node Maintenance Operator now validates maintenance requests
 
 The Node Maintenance Operator now validates maintenance requests for master nodes, preventing master (etcd) quorum violation. As a result, master nodes can only be set into maintenance if the `etcd-quorum-guard` pod disruption budget (PDB) allows it. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1826914[*BZ#1826914*])
 
+[id="ocp-4-6-node-maintenance-operator-cr"]
+==== Set nodes to maintenance mode using NodeMaintenance custom resources
+
+Previously, putting a node into maintenance mode using a `NodeMaintenance` custom resource was not documented. This is now fixed. For more information, see xref:../virt/node_maintenance/virt-setting-node-maintenance.adoc#virt-setting-node-to-maintenance-mode-with-cr_virt-setting-node-maintenance[Setting a node to maintenance mode with a NodeMaintenance custom resource]
```

Comment:
Suggested edit: 

s/custom resource/custom resource (CR)

The link works, but it's directed to the top of the assembly. Just want to make sure that is intentional. :)